pkg/util/kubernetes/kubelet/containers.go

```diff
@@ -96,6 +101,11 @@ func (ku *KubeUtil) getContainerMetrics(ctn *containers.Container) {
 	}
 	ctn.SetMetrics(metrics)
 
+	ctn.Pids, err = providers.ContainerImpl().GetPIDs(ctn.ID)
+	if err != nil {
+		log.Debugf("ContainerImplementation cannot get PIDs for container %s, err: %s", ctn.ID[:12], err)
+	}
+
```

Comment:
why do we return if we cannot get container metrics or network metrics but we continue if we cannot get pids?

Reply:
Aligned the behaviour and also `docker/containers.go`